openassessment/assessment/api/staff.py

```diff
@@ -26,7 +26,7 @@
 STAFF_TYPE = "ST"
 
 
-def submitter_is_finished(submission_uuid, requirements):
+def submitter_is_finished(submission_uuid, staff_requirements):
     """
     Determine if the submitter has finished their requirements for staff
     assessment. Always returns True.
```

Comment:
This function is called polymorphically here, so all the api types need to accept the same arguments in their method signature.

https://github.com/edx/edx-ora2/blob/efischer/enable_staff_grading/openassessment/workflow/models.py#L663

I'll get the docstring updated though.